utils/constant.js

```diff
@@ -0,0 +1,3 @@
+const userRole = 'user'
+const adminRole = 'admin'
+module.exports = {userRole,adminRole}
```

Comment:
variables in Constant.js should be in uppercase like ROLE_ADMIN, ROLE_USER

Reply:
Sure, I will make the changes accordingly.